docs_zh_CN/manual/cluster-admin/troubleshooting.md

```diff
@@ -61,6 +61,24 @@ pai_node_count{disk_pressure="false",instance="10.0.0.1:9101",job="pai_serivce_e
   2. 检查docker cache。
   3. 检查PAI的log文件夹：`/var/log/pai`.
 
+
+### NodeGpuCountChanged Alert
+
+这是来自alert manager的报警，用来监控各个机器上的GPU数量。
+如果机器上的GPU数量和`<config-dir>/layout.yaml`中声明的不符，就会有这个报警。
+
+如果您发现，GPU的数量是正确的，但这个报警还一直在，就有可能是`layout.yaml`声明错误导致的。
+
+解决方法：
+
+  1. 进入dev box容器，检查`layout.yaml`文件，修改GPU数量。
+  2. 更新配置，重启服务：
+    ```shell
+    /pai/paictl.py service stop -n cluster-configuration job-exporter
+    /pai/paictl.py config push -p <config-dir> -m service
+    /pai/paictl.py service start -n cluster-configuration job-exporter
+    ```
+
```

Comment:
There is a format issue [here](https://github.com/microsoft/pai/blob/master/docs/manual/cluster-admin/troubleshooting.md#nodegpucountchanged-alert), could you help to fix that ? For both en and cn versions.

Reply:
done